fapolicy_analyzer/ui/profiler_page.py

```diff
@@ -54,27 +56,27 @@ def __init__(self, fapd_manager):
             "start": [
```

Comment:
We should consider adding all the analyzer related buttons to the same key in the `actions`. The top level keys here are used for grouping related buttons. If the keys are different then a divider is shown in the toolbar to signify a different group of buttons. Instead of "start", "stop" and "analyze" I'd suggest "profiler" or simular.

Reply:
10-4, updated